examples/vector_databases/redis/redis-hybrid-query-examples.ipynb

```diff
@@ -0,0 +1,862 @@
+    "                                            model=\"text-embedding-ada-002\",\n",
+    "                                            )[\"data\"][0]['embedding']\n",
+    "\n",
+    "    # Prepare the Query\n",
+    "    base_query = f'{hybrid_fields}=>[KNN {k} @{vector_field} $vector AS vector_score]'\n",
+    "    query = (\n",
+    "        Query(base_query)\n",
+    "         .return_fields(*return_fields)\n",
+    "         .sort_by(\"vector_score\")\n",
+    "         .paging(0, k)\n",
+    "         .dialect(2)\n",
+    "    )\n",
+    "    params_dict = {\"vector\": np.array(embedded_query).astype(dtype=np.float32).tobytes()}\n",
+    "   \n",
+    "    # perform vector search\n",
+    "    results = redis_client.ft(index_name).search(query, params_dict)\n",
+    "    if print_results:\n",
+    "        for i, product in enumerate(results.docs):\n",
+    "            score = 1 - float(product.vector_score)\n",
+    "            print(f\"{i}. {product.productDisplayName} (Score: {round(score ,3) })\")\n",
+    "    return results.docs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "7e2025f6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0. John Players Men Blue Jeans (Score: 0.791)\n",
+      "1. Lee Men Tino Blue Jeans (Score: 0.775)\n",
+      "2. Peter England Men Party Blue Jeans (Score: 0.763)\n",
+      "3. Lee Men Blue Chicago Fit Jeans (Score: 0.761)\n",
+      "4. Lee Men Blue Chicago Fit Jeans (Score: 0.761)\n",
+      "5. French Connection Men Blue Jeans (Score: 0.74)\n",
+      "6. Locomotive Men Washed Blue Jeans (Score: 0.739)\n",
+      "7. Locomotive Men Washed Blue Jeans (Score: 0.739)\n",
+      "8. Do U Speak Green Men Blue Shorts (Score: 0.736)\n",
+      "9. Palm Tree Kids Boy Washed Blue Jeans (Score: 0.732)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# For using OpenAI to generate query embedding\n",
+    "results = search_redis(redis_client, 'man blue jeans', k=10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "93c4a696",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0. John Players Men Blue Jeans (Score: 0.739)\n",
+      "1. Lee Men Tino Blue Jeans (Score: 0.72)\n",
+      "2. Peter England Men Party Blue Jeans (Score: 0.718)\n",
+      "3. Denizen Women Blue Jeans (Score: 0.715)\n",
+      "4. Jealous 21 Women Washed Blue Jeans (Score: 0.708)\n",
+      "5. Jealous 21 Women Washed Blue Jeans (Score: 0.708)\n",
+      "6. Levis Kids Blue Solid Jean (Score: 0.706)\n",
+      "7. French Connection Men Blue Jeans (Score: 0.705)\n",
+      "8. Lee Men Blue Chicago Fit Jeans (Score: 0.705)\n",
+      "9. Lee Men Blue Chicago Fit Jeans (Score: 0.705)\n"
+     ]
+    }
+   ],
+   "source": [
+    "results = search_redis(redis_client, 'blue jeans', vector_field='product_vector', k=10)"
```

Comment:
do you need to declare the `vector_field` here? I think it's fine as long as it's consistent between calls. Side note, let's also make sure to format the blocks of code in the same way. This will get a ton of impressions from developers via openai